src/rgw/driver/sfs/sfs_lc.h

```diff
@@ -0,0 +1,75 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t
+// vim: ts=8 sw=2 smarttab ft=cpp
+/*
+ * Ceph - scalable distributed file system
+ * SFS SAL implementation
+ *
+ * Copyright (C) 2023 SUSE LLC
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation. See file COPYING.
+ */
+#pragma once
+
+#include "rgw_lc.h"
+#include "rgw_sal.h"
+#include "rgw_sal_sfs.h"
+
+#include <chrono>
+#include <mutex>
+
+namespace rgw::sal::sfs {
+
+class LCSFSSerializer : public StoreLCSerializer {
+  std::timed_mutex & mutex;
+
+public:
+  LCSFSSerializer(std::timed_mutex & m,
+                  SFStore* store, const std::string& oid,
+                  const std::string& lock_name,
+                  const std::string& cookie)
+                  : mutex(m) {}
+
+  virtual int try_lock(const DoutPrefixProvider *dpp, utime_t dur,
+                       optional_yield y) override {
+    if (mutex.try_lock_for(std::chrono::seconds(dur))) return 0;
```

Comment:
I know this is a style preference, but it makes readability quite hard. Mind moving the return to the line below?

Reply:
changed